src/Tools/dotnet-dump/ExtensionCommands/ClrMDHelper.cs

```diff
@@ -273,10 +273,749 @@ private string BuildTimerCallbackMethodName(ClrObject timerCallback)
             }
         }
 
+        public IEnumerable<ThreadPoolItem> EnumerateGlobalThreadPoolItems()
+        {
+            ClrModule mscorlib = GetMscorlib();
+            return IsNetCore() ?
+                EnumerateGlobalThreadPoolItemsInNetCore(mscorlib) :
+                EnumerateGlobalThreadPoolItemsInNetFramework(mscorlib);
+        }
+
+        public IEnumerable<ThreadPoolItem> EnumerateLocalThreadPoolItems()
+        {
+            ClrModule mscorlib = GetMscorlib();
+            return IsNetCore() ?
+                EnumerateLocalThreadPoolItemsInNetCore(mscorlib) :
+                EnumerateLocalThreadPoolItemsInNetFramework(mscorlib);
+        }
+
+        // here is the code to enumerate thread pool items from ThreadPool.cs
```

Comment:
Is there a reason this code is commented out?  We don't usually check in commented out code.

Reply:
This is the code in the CLR to list enumerate the thread pool items so it is used as documentation but I can remove it and provide a link to it (that might get broken in the future)